packages/cli/generators/app/templates/src/controllers/ping.controller.ts.ejs

```diff
@@ -10,6 +10,7 @@ const PING_RESPONSE: ResponseObject = {
     'application/json': {
       schema: {
         type: 'object',
+        title: 'loopback.PingResponse',
```

Comment:
I guess `PingResponse` should be good enough as it will be part of the application code - not a built-in schema from LoopBack.

Reply:
Ok, now it's `PingResponse`.